### PR TITLE
[Bugfix] fskip of EliminateCommonSubexpr cannot always return false

### PR DESCRIPTION
**Pull request by yuliujq**
Originally, fskip returns false at the end of PackedFunc, discards return true in 'cast' case

### CODE REVIEW

**Comment by yuliujq** [comment]
A little fix for fskip in EliminateCommonSubexpr, it will not return false in 'cast' case.

**Comment by vinx13** [comment]
could you take a look at the ci error?

**Comment by yzhliu** [comment]
also can we have a test case?

File: src/relay/backend/build_module.cc [hunk 1]
```diff
@@ -318,6 +318,7 @@ class RelayBuildModule : public runtime::ModuleNode {
     pass_seqs.push_back(transform::SimplifyInference());
     PackedFunc fskip = PackedFunc([](TVMArgs args, TVMRetValue* rv) {
       Expr expr = args[0];
+      *rv = false;
```
Comment (yuliujq) [inline src/relay/backend/build_module.cc]: Default return false

**Pull request merged**